packages/@aws-cdk/aws-stepfunctions-tasks/lib/dynamodb/shared-types.ts

```diff
@@ -0,0 +1,251 @@
+import { transformAttributeValueMap } from './private/utils';
+
+/**
+ * Determines the level of detail about provisioned throughput consumption that is returned.
+ */
+export enum DynamoConsumedCapacity {
```

Comment:
I wonder if those should be directly in the `dynamodb` package? I don't feel strongly either way, but it sounds like something that could be useful in other contexts using DDB tables?

Reply:
I think that makes sense. I'm not strongly opinionated here either.
I'll put together another PR moving them over and let's see what that looks like in a draft to continue with this idea.